.gitignore

```diff
@@ -11,4 +11,7 @@ cscope.*
 /src/tags
 # other temp files
 *.swp
-/src/support/logger/rust_bindings/Cargo.lock
\ No newline at end of file
+/src/support/logger/rust_bindings/Cargo.lock
+target
+.vscode
```

Comment:
`.vscode` is already included a few lines up.

Reply:
Oops, fixed